File: web_mes/src/models/Location/Address.js
class AddressModel{
    isSelected = false;
    constructor({id, name, gender, phone, province, district, ward, street}){
        this.id = id;
        this.name = name;
        this.gender = gender;
        this.phone = phone;
        this.province = province;
        this.district = district;
        this.ward = ward;
        this.street = street;
    }
}
export default AddressModel;

File: GolangQuest/internal/address/entities/address.go
package entities

import "gorm.io/gorm"

type Address struct {
	gorm.Model
	UserID       *string `gorm:"column:user_id"`
	Name         *string `gorm:"column:name"`
	Gender       *bool   `gorm:"column:gender"`
	Phone        *string `gorm:"column:phone"`
	ProvinceCode *string `gorm:"column:province_code"`
	DistrictCode *string `gorm:"column:district_code"`
	WardCode     *string `gorm:"column:ward_code"`
	Street       *string `gorm:"column:street"`
}

func (Address) TableName() string {
	return "Address"
}
